controllers/virtualmachinewebconsolerequest/v1alpha1/webconsolerequest_unit_test.go

```diff
@@ -141,5 +145,72 @@ func unitTestsReconcile() {
 			// Checking the label key only because UID will not be set to a resource during unit test.
 			Expect(wcrCtx.WebConsoleRequest.Labels).To(HaveKey(webconsolerequest.UUIDLabelKey))
 		})
+
+		When("SimplifiedEnablement Feature is true", func() {
+			JustBeforeEach(func() {
+				reconciler = webconsolerequest.NewReconciler(
+					pkgcfg.UpdateContext(
+						ctx,
+						func(config *pkgcfg.Config) {
+							config.Features.SimplifiedEnablement = true
+						},
+					),
+					ctx.Client,
+					ctx.Logger,
+					ctx.Recorder,
+					ctx.VMProvider,
+				)
+			})
+
+			scenarios := []struct {
```

Comment:
If I set the FSS in each iteration (in the same newreconciler call as the API Server DNS Name setting), then it will not actually set the FSS properly in fail. Hence, I do it up above.

Reply:
The problem that I am facing is that I am trying to modify the config (reconciler) just before each entry. It seems that JustBeforeEach and DescribeTable are not compatible with each other, so whenever I try to update proxySvcDNS within each Entry, the changes aren't actually going in affect.

Reply:
Discussed separately, added a describe table.